monkey/infection_monkey/monkey.py

```diff
@@ -86,12 +86,24 @@ def initialize(self):
         self._network = NetworkScanner()
         self._dropper_path = sys.argv[0]
 
+        server_num=0     
+        secondary_servers= len(self._args) 
+  
         if self._default_server:
-            if self._default_server not in WormConfiguration.command_servers:
-                LOG.debug("Added default server: %s" % self._default_server)
-                WormConfiguration.command_servers.insert(0, self._default_server)
-            else:
-                LOG.debug("Default server: %s is already in command servers list" % self._default_server)
+           if self._default_server[0] not in WormConfiguration.command_servers:
```

Comment:
This variable needs to be renamed to `default_servers`. Also, what does this `if` achieve?

Reply:
Won't it break other fn which are using default_server as an attribute...?